### ISSUE TITLE
LPub3D "Ignore ending without ignore begin" message

### ISSUE THREAD

**Issue by trevorsandy** [trevorsandy/lpub3d#80]
### Subject
After installing it successfully on my Mac, having updated the required libraries with Homebrew, I tried to load my latest ldr file, and had a bunch of errors... mostly "ignore ending without ignore begin", which I do not understand as it is correctly formed, and was perfectly working with the previous version... nested callouts warning too, and most annoying: the rotation of all the assemblies was messed up, whereas there was no modification at all on the ldr file... perhaps something to do with the 3D editor?

### Environment
* Version of LPub3D - v2.3.0
* Operating system - All

### Expected behaviour
Successful file load

### Actual behaviour
As reported above

### Workaround
None known

### Solution suggestion
Achieve expected behaviour

**Comment by trevorsandy** [comment]
I could not reproduce this behaviour with some of the more complex model files I have. Awaiting and example model file from the reporter.

**Comment by Davdup** [comment]
Hi Trevor!
Just sent you a message on SourceForge, with a link to the file... Hoping it will help!
Cheers

David

**Comment by trevorsandy** [comment]
Corrected.

Before v2.3.0, most of the meta parse errors, while they were captured, were benignly flashed in the status bar. This happened so quickly most users didn't even know this was the case. However,  With v2.3.0, I moved all the parse errors to a message dialog by flagging them as actual errors (versus warnings for example). All errors present to the user via message dialog if in UI mode or standard output when in batch mode so this is the reason for the increased number of error messages.

In addition, there was a condition bug (supposed to be "not true" but the "not" was missing) and this  falsely triggered MLCad and LSynth Ignore errors - this has been fixed.

With 2.3.1, I've corrected the condition error and softened the error message display, allowing users the choice of display or ignore which can be set in the Preferences dialogue:

![screenshot - 07_09_2018 18_19_17](https://user-images.githubusercontent.com/13388970/45233443-0b2ad480-b2d3-11e8-9eb4-61ea0202c1c9.png)

When Model Meta Parse Errors is checked, the user will now see a dialogue (when in UI mode) displaying the message along with the option to not display parse error messages in the future.

![screenshot - 07_09_2018 18_20_28](https://user-images.githubusercontent.com/13388970/45233543-5e9d2280-b2d3-11e8-8f00-73b69615611f.png)

With the condition bug fix, running LPub3D with Model Meta Parse Errors checked should not generate excessive error messages.

Cheers,
